api/proto/disperser/v2/disperser_v2.proto

```diff
@@ -141,6 +164,30 @@ message GetPaymentStateReply {
   bytes onchain_cumulative_payment = 5;
 }
 
+// GetQuorumSpecificPaymentStateReply contains the payment state of an account.
+message GetQuorumSpecificPaymentStateReply {
+  // payment vault parameters with per-quorum configurations
+  PaymentVaultParams payment_vault_params = 1;
+  // off-chain account reservation usage records
+  map<uint32, PeriodRecord> period_records = 2;
```

Comment:
what is this uint32? Guessing its a quorum number. needs to be properly documented

Reply:
added documentation

Reply:
please add commit links, makes the reviewers' job much much easier (and the PR author's job not that much more harder.. take 15 seconds more)